Cargo.toml

```diff
@@ -128,6 +128,8 @@ prost = { version = "0.12" }
 icelake = { git = "https://github.com/icelake-io/icelake", rev = "54fd72fbd1dd8c592f05eeeb79223c8a6a33c297", features = [
     "prometheus",
 ] }
+iceberg = { git = "https://github.com/risingwavelabs/iceberg-rust.git", rev = "e6ae6229dfd0c0a8793cde42a7d626c704ea9088" }
+iceberg-catalog-rest = { git = "https://github.com/risingwavelabs/iceberg-rust.git", rev = "e6ae6229dfd0c0a8793cde42a7d626c704ea9088" }
```

Comment:
Any reason why we need to use a forked version?

Reply:
for now, the upstream reader is easy to cause redundant. detail https://github.com/apache/iceberg-rust/issues/398
We can refactor to use the upstream interface when the upstream PR merge. https://github.com/apache/iceberg-rust/pull/401

Reply:
Can you comment the reason for fork in Cargo.toml?